tag_reader.go

```diff
@@ -71,12 +74,37 @@ func (r *tagReader) ReadAll() ([]*Tag, error) {
 		})
 	}
 
-	r.sortTags(tags)
-	r.assignPreviousAndNextTag(tags)
+	switch r.sortBy {
+	case "date":
+		r.sortTags(tags)
+		r.assignPreviousAndNextTag(tags)
```

Comment:
Maybe move `r.assignPreviousAndNextTag(tags)` outside of switch, because it is a shared operation for all.

Reply:
good one.